org.OpenT2T.Sample.SuperPopular.Thermostat/org.OpenT2T.Sample.SuperPopular.Thermostat.xml

```diff
@@ -10,10 +10,20 @@
     <method name="turnOff"/>
     
     <!-- Get the current ambient temperature -->
-    <method name="getCurrentTemperature">
+    <method name="getAmbientTemperature">
```

Comment:
But can you accommodate reading both C/F units in it and when I merge I will hook the 'scale/unit' args to it.

---
In reply to: [68288119](https://github.com/openT2T/translators/pull/22#discussion_r68288119) [](ancestors = 68288119,68286096)